tools/filters/blockquote2div.py

```diff
@@ -52,6 +52,7 @@
     "callout": ("panel-info", "glyphicon-pushpin"),
     "challenge": ("panel-success", "glyphicon-pencil"),
     "prereq": ("panel-warning", "glyphicon-education"),
+    "getready": ("panel-warning", "glyphicon-check"),
```

Comment:
Good! We need a warning and not an error as some lessons (e.g. `git-novice`) don't need extra files.